modules/backend/controllers/Auth.php

```diff
@@ -100,12 +100,19 @@ public function signin_onSubmit()
             'password' => post('password')
         ], $remember);
 
-        try {
-            // Load version updates
-            UpdateManager::instance()->update();
+
+        $runMigrationsOnLogin = Config::get('cms.runMigrationsOnLogin', null);
+        if ($runMigrationsOnLogin === null) {
```

Comment:
use is_null here instead please

Reply:
I used same structure like is used on other "null" config values.